src/Components/Nav/Nav.js

```diff
@@ -1,7 +1,125 @@
-import React from 'react';
+import { Link } from 'react-router-dom';
+
+import styled from 'styled-components';
 
 function Nav() {
-  return <div></div>;
+  return (
+    <Navigator>
+      <NavWarpper>
+        <Link to="/">
+          <Logo>iltal</Logo>
+        </Link>
+        <TravelAndActivity>
+          <MainNav>
+            <Link>여행</Link>
+            <SubNav>
+              <li>
+                {TRAVEL_LSIT.map((nav, index) => (
```

Comment:
```suggestion
                {TRAVEL_LIST.map((nav, index) => (
```
오타 전부 확인해서 고쳐주세요! 현재 에러는 안나는 상황이지만 이런 오타는 언제든지 휴먼 에러를 발생시킬 수 있으므로 발견할 때마다 잘 고쳐두어야 합니다.

Reply:
수정 완료하였습니다. 꼼꼼하게 체크해주셔서 감사합니다 종택님!! 👍🏻